tests/SideBySide/CancelTests.cs

```diff
@@ -228,6 +228,18 @@ public void DapperQueryMultiple()
 			}
 		}
 
+		[Fact]
+		public async Task CancelCompletedCommand()
+		{
+			using (var cmd = m_database.Connection.CreateCommand())
+			{
+				cmd.CommandText = @"select 1;";
+
+				using (await cmd.ExecuteReaderAsync().ConfigureAwait(false))
+					cmd.Cancel();
+			}
```

Comment:
I tried `SELECT 2;` but that doesn't repro the problem, as per bullet 2 of [bug 45679](https://bugs.mysql.com/bug.php?id=45679). So I think the `INSERT`/`UPDATE` combination of commands is a good one.